OpenTrons/opentrons-api
# feat(app): add appMode and robotType as new Mixpanel properties

# Overview
Add new 2 properties to Mixpanel.
appMode: `Desktop` or `ODD`
appMode will be sent always

robotType: `OT-2 Standard` or `OT-3 Standard`
robotType will be sent when a user does something related protocol run event such as runPause etc


# Test Plan
- run app with `OT_APP_MIXPANEL_ID`
- do something with Desktop app or ODD app (on-device-mode)
- check Mixpanel



# Changelog
- update epic.ts and mixpanel.ts
- update `parseProtocolRunAnalyticsData`


# Review requests



# Risk assessment
low
